src/conn/conn_dhandle.c

```diff
@@ -847,6 +847,33 @@ __wt_conn_dhandle_discard(WT_SESSION_IMPL *session)
     return (ret);
 }
 
+/*
+ * __wt_dhandle_update_write_gens --
+ *     Update the open dhandles write generation and base write generation number.
+ */
+void
+__wt_dhandle_update_write_gens(WT_SESSION_IMPL *session)
+{
+    WT_BTREE *btree;
+    WT_CONNECTION_IMPL *conn;
+    WT_DATA_HANDLE *dhandle;
+
+    conn = S2C(session);
+
+    for (dhandle = NULL;;) {
+        WT_WITH_HANDLE_LIST_WRITE_LOCK(session, WT_DHANDLE_NEXT(session, dhandle, &conn->dhqh, q));
+        if (dhandle == NULL)
+            break;
+        btree = (WT_BTREE *)dhandle->handle;
+
+        WT_ASSERT(session, btree != NULL);
+
+        /* Initialize the btrees write gens to connection base write gen */
```

Comment:
My bad, but this comment should be amended.

Reply:
Comment updated.